src/main/java/mate/academy/spring/dao/impl/MovieSessionDaoImpl.java

```diff
@@ -57,4 +58,50 @@ public Optional<MovieSession> get(Long id) {
             throw new DataProcessingException("Can't get a movie session by id: " + id, e);
         }
     }
+
+    @Override
+    public void delete(Long id) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = sessionFactory.openSession();
+            transaction = session.beginTransaction();
+            session.createQuery("delete from MovieSession where id=:movieSessionId")
+                    .setParameter("movieSessionId", id)
```

Comment:
```suggestion
                    .setParameter("id", id)
```
We already know that we work with movie sessions)